dapps/src/tests/helpers.rs

```diff
@@ -58,22 +60,35 @@ impl ContractClient for FakeRegistrar {
 	}
 }
 
+fn init_logger() {
+	// Initialize logger
+	if env::var("RUST_LOG").is_ok() {
+		let mut builder = LogBuilder::new();
+		builder.parse(&env::var("RUST_LOG").unwrap());
+		builder.init().unwrap();
```

Comment:
`is_ok` -> `unwrap` is better done with `if let Ok(...)`.

what's the guarantee that the `RUST_LOG` flags are valid? edit: well, if it's only used in test code, the unwrap is alright :)